### ISSUE TITLE
Dexy-Color-Dark-Gtk doesn't apply

### ISSUE THREAD

**Issue by RosGtHub** [L4ki/Dexy-Plasma-Themes#2]
![image](https://github.com/L4ki/Dexy-Plasma-Themes/assets/89165766/91fe5f69-a455-4058-a8ea-162c2ce1fe54)

**Comment by L4ki** [comment]
Hi  RosGtHub,

I think that's fine. It looks like this for me:

 [![Screenshot-20230816-142800.png](https://i.postimg.cc/fTHHJHz6/Screenshot-20230816-142800.png)](https://postimg.cc/VdC9hWbR)

**Comment by RosGtHub** [comment]
shouldn't it have the title bar with the three theme buttons? is it because i've  wings-dark theme? I wanted gtk applications to have more consistent buttons so I decided to replace the "breeze" plasma style with dexy but didn't work.

**Comment by L4ki** [comment]
Yes, it refers to Window Decorations. I'll see about putting it in the GTK Theme. I'll let you know when I do.

**Comment by RosGtHub** [comment]
alright
